.github/workflows/test-publish.yml

```diff
@@ -36,9 +38,9 @@ jobs:
           tag: ${{ github.event.inputs.version }}
           commit: main
 
-      - name: deploy
+      - name: Publish to TestPyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           user: __token__
-          password: ${{ secrets.PYPI }}
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
```

Comment:
We don't have a test PYPI token yet -- but also this step looks different than the one from the prod release?